apps/src/music/blockly/blocks/control.js

```diff
@@ -1,57 +1,9 @@
 import {BlockTypes} from '../blockTypes';
 
-export const loopFromTo = {
-  definition: {
-    type: BlockTypes.LOOP_FROM_TO,
-    style: 'loop_blocks',
-    message0: 'loop %1 from %2 to %3',
-    args0: [
-      {
-        type: 'field_variable',
-        name: 'measure',
-        variable: 'measure',
-      },
-      {
-        type: 'field_number',
-        name: 'from',
-        value: 1,
-        min: 1,
-      },
-      {
-        type: 'field_number',
-        name: 'to',
-        value: 5,
-        min: 1,
-      },
-    ],
-    message1: 'do %1',
-    args1: [
-      {
-        type: 'input_statement',
-        name: 'code',
-      },
-    ],
-    inputsInline: true,
-    previousStatement: null,
-    nextStatement: null,
-    tooltip: 'loop from a number to another number',
-    helpUrl: '',
-  },
-  generator: ctx =>
-    'for (var measure = ' +
-    ctx.getFieldValue('from') +
-    '; measure <= ' +
-    ctx.getFieldValue('to') +
-    '; measure++) {\n' +
-    //ctx.getFieldValue('code') +
-    Blockly.JavaScript.statementToCode(ctx, 'code') +
-    '\n}\n',
-};
-
 export const forLoop = {
   definition: {
     type: BlockTypes.FOR_LOOP,
-    message0: 'for %1 from %2 to %3 by %4',
+    message0: '%{BKY_CONTROLS_FOR_TITLE}',
```

Comment:
Is this in scope without being explicitly imported?

Reply:
Yep, this gets handled by Blockly under the hood. Since this is nearly a direct copy of Blockly's for loop, I opted to use their translated string instead of handling the translation ourselves.